integration/kubernetes/helm-chart/alluxio/templates/master/journal-pv.yaml

```diff
@@ -9,17 +9,22 @@
 # See the NOTICE file distributed with this work for information regarding copyright ownership.
 #
 
+{{ $root := . -}}
+{{- $masterCount := int .Values.master.count }}
+{{- range $i := until $masterCount }}
```

Comment:
Now we need to manually generate 1 PV for each master Pod if it needs one for journal.

Reply:
i'm in favor of removing the pv file from the template and providing an example in documentation instead